src/utils.test.js

```diff
@@ -66,6 +66,17 @@ describe('utils', () => {
       {start: 9, end: 14, highlight: false}
     ])
   })
+  
+  it('should handle unclosed square brackets when autoEscape prop is truthy', () => {
+    const rawChunks = Chunks.findChunks({
+      autoEscape: true,
+      searchWords: ['text['],
+      textToHighlight: ']This is text['
+    })
+    expect(rawChunks).to.eql([
+      {start: 9, end: 14, highlight: false}
+    ])
+  })
```

Comment:
This test also passes on the main branch without the accompanied regex change. Is there a different test that we could add that captures the original bug?

Reply:
hmm, I'll look into it